<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSeparations</class>
 <widget class="QWidget" name="WdgSeparations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>392</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Current color model:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblColormodel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="grpSource">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Source</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioCurrentLayer">
          <property name="text">
           <string>Current layer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioAllLayers">
          <property name="text">
           <string>Flatten all layers before separation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpOutput">
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioLayers">
          <property name="text">
           <string>To layers</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioImages">
          <property name="text">
           <string>To images</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpAlpha">
     <property name="title">
      <string>Alpha Options</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioCopyAlpha">
        <property name="text">
         <string>Copy alpha channel to each separated channel as an alpha channel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioDiscardAlpha">
        <property name="text">
         <string>Discard alpha channel</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioSeparateAlpha">
        <property name="text">
         <string>Create separate separation from alpha channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkDownscale">
     <property name="text">
      <string>Downscale to 8-bit before separating</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkColors">
     <property name="text">
      <string>Output to color, not grayscale</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioCurrentLayer</tabstop>
  <tabstop>radioLayers</tabstop>
  <tabstop>radioDiscardAlpha</tabstop>
  <tabstop>chkDownscale</tabstop>
  <tabstop>chkColors</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
